app/helpers/review_mapping_helper.rb

```diff
@@ -0,0 +1,123 @@
+    bgcolor = line_num.even? ? "#ffffff" : "#DDDDBB"
+    (1..@assignment.num_review_rounds).each {|round| instance_variable_set("@review_in_round_" + round.to_s, 0) }
+
+    response_maps = ResponseMap.where(["reviewed_object_id = ? AND reviewer_id = ? AND type = ?", reviewed_object_id, reviewer_id, type])
+    response_maps.each do |ri|
+      rspan += 1 if Team.exists?(id: ri.reviewee_id)
+      responses = ri.response
+      (1..@assignment.num_review_rounds).each do |round|
+        instance_variable_set("@review_in_round_" + round.to_s, instance_variable_get("@review_in_round_" + round.to_s) + 1) if responses.exists?(round: round)
+      end
+    end
+    [response_maps, bgcolor, rspan, line_num]
+  end
+
+  def get_team_reviewed_link_name(max_team_size, response, reviewee_id)
+    team_reviewed_link_name = if max_team_size == 1
+                                TeamsUser.where(team_id: reviewee_id).first.user.fullname
+                              else
+                                Team.find(reviewee_id).name
+                              end
+    team_reviewed_link_name = "(" + team_reviewed_link_name + ")" if !response.empty? and !response.last.is_submitted?
+    team_reviewed_link_name
+  end
+
+  def get_current_round_for_review_report(reviewer_id)
+    user_id = Participant.find(reviewer_id).user.id
+    topic_id = SignedUpTeam.topic_id(@assignment.id, user_id)
+    current_round = @assignment.number_of_current_round(topic_id)
+    current_round = @assignment.num_review_rounds if @assignment.get_current_stage(topic_id) == "Finished" || @assignment.get_current_stage(topic_id) == "metareview"
+  end
+
+  # varying rubric by round
+  def get_each_round_score_awarded_for_review_report(reviewer_id, team_id)
+    (1..@assignment.num_review_rounds).each {|round| instance_variable_set("@score_awarded_round_" + round.to_s, '-----') }
+    (1..@assignment.num_review_rounds).each do |round|
+      if @review_scores[reviewer_id] && @review_scores[reviewer_id][round] && @review_scores[reviewer_id][round][team_id] && @review_scores[reviewer_id][round][team_id] != -1.0
+        instance_variable_set("@score_awarded_round_" + round.to_s, @review_scores[reviewer_id][round][team_id].inspect + '%')
+      end
+    end
+  end
+
+  def get_min_max_avg_value_for_review_report(round, team_id)
+    [:max, :min, :avg].each {|metric| instance_variable_set('@' + metric.to_s, '-----') }
+    if @avg_and_ranges[team_id] && @avg_and_ranges[team_id][round] && [:max, :min, :avg].all? {|k| @avg_and_ranges[team_id][round].key? k }
+      [:max, :min, :avg].each do |metric|
+        metric_value = @avg_and_ranges[team_id][round][metric].nil? ? '-----' : @avg_and_ranges[team_id][round][metric].round(0).to_s + '%'
+        instance_variable_set('@' + metric.to_s, metric_value)
+      end
+    end
+  end
+
+  #
+  # for author feedback report
+  #
+  #
+  # varying rubric by round
+  def get_each_round_review_and_feedback_response_map_for_feedback_report(author)
```

Comment:
Assignment Branch Condition size for get_each_round_review_and_feedback_response_map_for_feedback_report is too high. [19.67/15]